frontend/src/pages/myTask/MyTask.jsx

```diff
@@ -1,7 +1,190 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from '@mui/material';
+import { CheckCircle, RadioButtonUnchecked, Flag } from '@mui/icons-material';
+import task from '../../assets/images/Component 4.svg';
+import swap from '../../assets/images/Swap.svg';
+import filter from '../../assets/images/Filter.svg';
+import './MyTask.css';
 
 const MyTask = () => {
-  return <div>Here is My task</div>;
+  const [boards] = useState([
+    {
+      id: 1,
+      title: 'Assigned',
+      tasks: [
+        {
+          id: 1,
+          Task: 'Design feedback on wireframe',
+          DueDate: '7 Oct',
+          Priority: 'Hard',
+          Objective: '3 Objective',
+          Status: 'Ongoing',
+          Project: 'Bookum App',
+        },
+        {
+          id: 2,
+          Task: 'Moodboarding',
+          DueDate: '7 Oct',
+          Priority: 'Hard',
+          Objective: '3 Objective',
```

Comment:
Shouldn't you set it up to use real data by now? If you want to have test data, you can setup a dummy backend or make it easy to switch between real and fake data by using a bool